### DISCUSSION TITLE
Search for variables lists or/and messages

### DISCUSSION THREAD

**Discussion by gometrysten** [ScratchAddons/ScratchAddons#1840] [Ideas] [answered]
We all experienced scrolling through all of your variables, just to switch to another variable. Typing in it's name in a search bar would be much more efficient.

**Comment by mxmou** [comment, marked as answer]
There is already a feature called searchable dropdowns that does this.
![Screenshot](https://scratchaddons.com/assets/images/addons/editor-searchable-dropdowns.png)
